doc/aurpublish.1.asciidoc

```diff
@@ -0,0 +1,83 @@
+aurpublish(1)
+=============
+
+Name
+----
+aurpublish - PKGBUILD management framework for the Arch User Repository
+
+
+Synopsis
+--------
+'aurpublish' [options] PACKAGE
+'aurpublish' log [<options>] PACKAGE
+'aurpublish' setup
+
+
+Description
+-----------
+Aurpublish is an utility which helps maintainers of aur packages.
+
+
+How it works
+------------
+Commit PKGBUILDs in named subdirectories.
+Export them to the AUR with the aurpublish command, using the subtree push 
+stratagem. This preserves an independent history for third-party hosting, 
+pull requests... ;)
+
+Commands
+--------
+
+setup::
```

Comment:
Should be bolded by adding asterisks around it.
```suggestion
*setup*::
```

Reply:
Fixed in https://github.com/eli-schwartz/aurpublish/pull/16/commits/1f859dabd05002408ee47a870c34bf9311d607b3.

Reply:
Fixed in https://github.com/eli-schwartz/aurpublish/pull/16/commits/1f859dabd05002408ee47a870c34bf9311d607b3.

Reply:
Fixed in https://github.com/eli-schwartz/aurpublish/pull/16/commits/1f859dabd05002408ee47a870c34bf9311d607b3.